tests/integration/mobile_app/upload.rs

```diff
@@ -223,7 +224,8 @@ fn command_mobile_app_upload_ipa_chunked() {
                 } else {
                     r#"{
                         "state": "ok",
-                        "missingChunks": []
+                        "missingChunks": [],
+                        "artifactId": "some-text-id"
```

Comment:
just wondering, is it realistic that we would have a text ID?

Reply:
Yes, we are hopefully soon going to change the public artifact ids to be UUID based, see: https://sentry.slack.com/archives/C1G612XHC/p1754417960518429